consul/acl_test.go

```diff
@@ -861,6 +862,73 @@ func TestACL_filterNodeDump(t *testing.T) {
 	}
 }
 
+func TestACL_filterPreparedQueries(t *testing.T) {
```

Comment:
Can you add a negative test in here to make sure we don't leak information back (e.g. `Token`)?